docs/types.rst

```diff
@@ -104,6 +104,12 @@ Operators:
 
 * ``<=``, ``<``, ``==``, ``!=``, ``>=`` and ``>``
 
+.. warning::
+  If you convert a type that uses a larger byte size to an ``address``, for example ``byte32``, then the ``address`` is truncated. To reduce conversion ambiguity version 0.4.24 and higher of the compiler force you make the truncation explicit in the conversion. Take for example the address ``0x111122223333444455556666777788889999AAAABBBBCCCCDDDDEEEEFFFFCCCC``.
+
+  You can use ``address(uint160(bytes20(b)))``, which results in 0x111122223333444455556666777788889999aAaa,
```

Comment:
Please also put these literals in double backticks.

Reply:
👍 Done.